packages/appstore/src/components/onboarding-new/static-dashboard.tsx

```diff
@@ -262,7 +262,7 @@ const StaticDashboard = ({
                                 <div className='static-dashboard-wrapper__body--apps-item'>
                                     <StaticTradingAppCard
                                         icon='DTrader'
-                                        name='DTrader'
+                                        name='Deriv Trader'
```

Comment:
Can this file be improved to utilise `brand.ts` from shared package? This way we won't need to change it again when there's the next rebranding.

Reply:
All of the icons used in this component are basically placed in one file which is used in the whole appstore package. I do not think if it is necessary to make changes to the component for this purpose.

Reply:
What I mean is the label, can we utilise the config from shared package instead, so there wouldn't be repetitive changes if the product name needs to be updated again?